tools/ci_build/github/azure-pipelines/linux-migraphx-ci-pipeline.yml

```diff
@@ -0,0 +1,54 @@
+trigger: none
+
+name: 'linux_ci_$(Date:yyyyMMdd)_$(Rev:r)'
+jobs:
+- job: AMDMIGraphX_CI
+  pool: 'AMD-GPU'
+  timeoutInMinutes: 60
+
+  # gid of video and render group on gcr-openpai-35 and -36
+  variables:
+    - name: video
+      value: 44
+    - name: render
+      value: 109
+
+  # generated from tools/ci_build/github/pai/migraphx-ci-pipeline-env.Dockerfile
+  container:
+    image: onnxruntimecibuildenvironment.azurecr.io/migraphx-ci-pipeline-env
```

Comment:
What's this? Who maintains this image?

Reply:
It's a fixed docker image for MIGraphX CI, it's like the "rocm-ci-pipeline-env" in https://github.com/microsoft/onnxruntime/blob/e3a2d5cca8bcefe064f83d57e46ea51ddb2b16e8/tools/ci_build/github/azure-pipelines/orttraining-pai-ci-pipeline.yml#L20

Reply:
Could you please create a different ACR for this?

Reply:
I don't know exactly what this ACR was created for, but now only AMD CI is using it.